install/kubernetes/cilium/values.yaml

```diff
@@ -530,6 +541,23 @@ hubble:
     # Port to listen to.
     listenPort: "4245"
 
+    # TLS configuration for Hubble Relay
+    tls:
+      # base64 encoded PEM values for the hubble-relay client certificate and private key
+      # This keypair is presented to Hubble server instances for mTLS authentication.
+      # These values need to be set manually if hubble.tls.auto.enabled is false.
```

Comment:
How about a simple sentence here to explain why there is no `enabled` field? Basically, the `hubble.relay.tls.client` stuff is enabled when `hubble.tls.enabled` is `true`. It just "feel" weird to me that `hubble.relay.tls.server.enabled` exist but there is no `hubble.relay.tls.client.enabled`.